lib/pure/smtp.nim

```diff
@@ -230,14 +225,32 @@ proc helo*(smtp: Smtp | AsyncSmtp) {.multisync.} =
   await smtp.debugSend("HELO " & smtp.address & "\c\L")
   await smtp.checkReply("250")
 
+proc recvEhlo*(smtp: Smtp | AsyncSmtp): Future[bool] {.multisync.} =
```

Comment:
do we need this to be exported? it's used inside `ehlo` so maybe better to be private?

Reply:
no you're right, i'll not export it